cryptoapi_windows.go

```diff
@@ -9,16 +9,60 @@ import (
 	"time"
 
 	"golang.org/x/sys/windows/registry"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var (
+	cryptoApiFlagGroup            = cflag.NewGroup(flagGroup, "capi")
+	cryptoApiFlagLogicalStoreName = cflag.String(cryptoApiFlagGroup, "store", "Root",
+		"Name of CryptoAPI logical store to inject certificate into. Consider: Root, Trust, CA, My")
+	cryptoApiFlagLogicalStoreScope = cflag.String(cryptoApiFlagGroup, "scope", "HKLM",
```

Comment:
The word "Logical" should not be in this variable's name, since HKLM/HKCU is part of the physical store, not the logical store.